native-maven-plugin/src/testFixtures/groovy/org/graalvm/buildtools/maven/AbstractGraalVMMavenFunctionalTest.groovy

```diff
@@ -49,12 +49,15 @@ import org.eclipse.jetty.server.handler.ResourceHandler
 import spock.lang.Specification
 import spock.lang.TempDir
 
+import java.nio.file.FileVisitResult
 import java.nio.file.Files
 import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
 import java.nio.file.StandardCopyOption
+import java.nio.file.attribute.BasicFileAttributes
 
 abstract class AbstractGraalVMMavenFunctionalTest extends Specification {
-    @TempDir
```

Comment:
Why you have removed @TempDir annotation and then proceeded with deleting files manually in the cleanup function?

Reply:
In my local when I work with Tempdir No alla tests pass, because the on windows the tempDir avoids white spaces in path with a certain encoding (C:\Users\Lahoucine~\AppData\Temp) so no parsing errors will show up if I have withe spaces in path.